src/pages/actions/reducer.ts

```diff
@@ -19,12 +19,21 @@ export type SideEffect =
     }
   | {
       type: "reloadData";
+    }
+  | {
+      type: "updateActionRecord";
```

Comment:
:upside_down_face: The term `record` is mentioned here the first time. I like it though! I think it would be good if we would align the language for the other parts aswell: thinking of something like `LedgerActionExecutionRecordStore`?

Reply:
Going with your fun idea and renaming this event type to `memoriseLedgerAction`.